### ISSUE TITLE
Change in <content> behavior between 3.4.3 and 3.4.4

### ISSUE THREAD

**Issue by stevenscreek** [NuGet/Home#3149]
Repro steps:
- Create a new class library project.
- Create a new nuspec via nuget spec & provide value for tokens
- Add content items to the project, such as:

```
<ItemGroup>
    <Content Include="README1.txt">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
    <Content Include="README2.txt">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```
- Add `<files />` to the nuspec file.
- Build the package with nuget pack Project.csproj.

Using 3.4.3.855, the `<files />` suppresses addition of the `<content>` items as described in #1177.
Using 3.4.4.1321, the `<content>` items are added to the output package.

As a result, we now have some enormous packages with the unintended content.

**Comment by emgarten** [comment]
@stevenscreek would you try the latest 3.5.0-beta2 and latest nightly from here to see if this is fixed in those builds: https://dist.nuget.org/index.html

**Comment by stevenscreek** [comment]
Both 3.5.0.1520 and 3.5.1.1597 show the behavior of 3.4.4 (including the content).

To be clear, as long as there's some way of excluding the <content> items, we're fine with changing how we accomplish that.

**Comment by emgarten** [comment]
Thanks for checking this sounds like a bug in pack.

**Comment by rrelyea** [comment]
Won't be able to address in 3.5, due to time restrictions.

**Comment by rohit21agrawal** [comment]
this has been fixed in 3.5 as a result of fixing https://github.com/NuGet/Home/issues/3257
